### ISSUE TITLE
RevealWidget - documentation

### ISSUE THREAD

**Issue by kookma** [Jermolene/TiddlyWiki5#3352]
https://tiddlywiki.com/#RevealWidget

After including the new parameter default, the description in the above tiddler should be corrected.
For example

> The reveal widget hides or shows its content depending upon the value of a state tiddler. The type of the widget determines the condition for the content being displayed:

I means the behavior when default value is set is not described.

**Comment by 00SS** [comment]
Also to fix:

The tiddler [RevealWidget.tid](https://github.com/Jermolene/TiddlyWiki5/blob/master/editions/tw5.com/tiddlers/widgets/RevealWidget.tid) refers to non-existent : StateTiddlers
The tiddler [StateMechanism.tid](https://github.com/Jermolene/TiddlyWiki5/blob/master/editions/tw5.com/tiddlers/mechanisms/StateMechanism.tid)  refers to non-existent : StateTiddler

A link should be in  StateMechanism.tid to : [[StateTiddler|StateTiddlers]]

The tiddler : StateTiddlers.tid should be newly created in [this Directory](https://github.com/Jermolene/TiddlyWiki5/tree/master/editions/tw5.com/tiddlers/concepts)
It should be tagged with: Concepts

It needs to briefly explain what is meant by a State Tiddler.
Something on the lines of:
`This is a tiddler that is used to hold the state of the user interface. Changes to the user interface are made indirectly, by changing the underlying state tiddlers, and letting ~TiddlyWiki ripple the changes through the user interface.`

**Comment by 00SS** [comment]
I find this whole documentation hard to follow. 

I would like to see an example each for Attributes:
 - default
 - stateTitle
 - stateField
 - position (also the default is not mentioned)

I tried to use stateTitle & stateField (v5.19) but couldn't get it to work. Don't know if that is related to #3874
